.github/ISSUE_TEMPLATE/rfc.yaml

```
@@ -0,0 +1,63 @@
+name: RFC
+description: Raise an RFC to discuss a significant change to the package that requires discussion and wider input.
```

How do we feel about the word 'package'?

```suggestion
description: Raise an RFC to discuss a significant change to the project that requires discussion and wider input.
```

---

In this repo it is a good suggestion. In monorepos it might make sense to say "a package".